challenge.md

```diff
@@ -31,21 +31,15 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+It is an absolute path. You can tell because it starts at the home directory, it gives a precise description of where the files lives. It wouldn't matter where we are because of how descriptive the path is.'
```

Comment:
Your answer is a little unclear at the beginning ("because it starts at the home directory" - do you mean "because the path starts at the root"?), but you are correct on all the major points.